### PR TITLE
Marketplace governance

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/en/docs/appstore/general/share-app-store-content.md [hunk 1]
```diff
@@ -44,7 +44,18 @@ These are some general guidelines for creating new Marketplace content:
...
 
-### 3.2 For Widgets
+### 3.2 Intellectual Property
+
+When considering intellectual property (IP) in the Mendix Marketplace, follow these guidelines:
+
+* Do not copy any text, names,or other data from other components published i the Marketplace, since you do not own the copyright. Do not create unnecessary ambiguity or confusion that would mislead consumers of Marketplace components.
+* Do not mention components published by others within your content on the Marketplace. 
+* Make sure your component documentation only includes intellectual property that you have created. Your component should not include information or content published by others on the Marketplace, as this may lead to removing your component from the Marketplace. This also means other components can also be removed if they are using your component information or content. 
+* If you come across instances of intellectual property abuse, let us know at someemail@mendix.com.
```
Comment (Adam-Dupaski) [inline content/en/docs/appstore/general/share-app-store-content.md]: What is the proper email to include here?

File: content/en/docs/appstore/general/share-app-store-content.md [hunk 2]
```diff
@@ -124,15 +134,18 @@ Follow these steps to describe your content:
...
+	* [GNU General Public License, version 3](https://www.gnu.org/licenses/gpl-3.0.en.html) – please note that everything licensed under GNU GPL is public and that GNU GPL has strong copyleft effect, which means that all consuming applications should make their code public as well; using this license is not recommended
+	* [Mendix EULA](https://www.mendix.com/terms-of-use/) – this license is to be used when the component is released by Mendix
 	* [MindSphere Development License Agreement](https://siemens.mindsphere.io/en/terms)
 	* [MIT](https://opensource.org/licenses/MIT)
 	* SDISW Development License Agreement for Mendix Sample Apps EXTERNAL
 	* SDISW End-User License Agreement for Mendix Connectors and Widgets
 	* SDISW End-User License Agreement for Mendix Connectors and Widgets EXTERNAL
 	* SDISW End-User License Agreement for Mendix Sample Apps
 
+	{{% alert color="info" %}}After you select a license, make sure you add a specific *license.txt* file in your artifacts (meaning, in the *.mpk*).
```
Comment (Adam-Dupaski) [inline content/en/docs/appstore/general/share-app-store-content.md]: Is this valid for all licenses, or just Apache V2 and MIT?

File: content/en/docs/appstore/general/share-app-store-content.md (hunk 1 above)
Comment (KseniiaIankina) [reply content/en/docs/appstore/general/share-app-store-content.md]: AppServices_Supplier_Team@mendix.com

File: content/en/docs/appstore/general/share-app-store-content.md [hunk 3]
```diff
@@ -256,7 +269,22 @@ To update content that has already been published, follow these steps:
...
+
+As the Mendix Marketplace grows, it is important for users to be able to find up-to-date and relevant components. In order to reduce the likelihood that users find outdated or obsolete components, we review Marketplace content and evaluate for the following points:
+
+* Whether the component supports the versions of Studio Pro that Mendix supports (meaning, the current major version plus two previous major versions – for more information, see [LTS, MTS & Monthly Releases](/releasenotes/studio-pro/lts-mts/))
+* Whether the component has been updated recently or not for ages (for example, it was published in 2016 and has not been updated since)
+* Whether it is being actively used or if it has limited usage or very few downloads, reviews, or ratings
+
+For a component that is outdated based on the above points, this is the review and remediation process: 
+
+1. Mendix sends a notification to the owner of the outdated component, who then has to submit an update within 30 days in order for their component to remain active on the Marketplace. This update needs to be based on support for an active version of Studio Pro.
```
Comment (KseniiaIankina) [inline content/en/docs/appstore/general/share-app-store-content.md:282]: Mendix sends a notification to the owner of the outdated component and company admin, who then..

File: content/en/docs/appstore/general/share-app-store-content.md [hunk 4]
```diff
@@ -124,15 +134,18 @@ Follow these steps to describe your content:
...
-	* [GNU General Public License, version 3](https://www.gnu.org/licenses/gpl-3.0.en.html)
-	* [Mendix EULA](https://www.mendix.com/terms-of-use/)
+	* [GNU General Public License, version 3](https://www.gnu.org/licenses/gpl-3.0.en.html) – please note that everything licensed under GNU GPL is public and that GNU GPL has strong copyleft effect, which means that all consuming applications should make their code public as well; using this license is not recommended
+	* [Mendix EULA](https://www.mendix.com/terms-of-use/) – this license is to be used when the component is released by Mendix
 	* [MindSphere Development License Agreement](https://siemens.mindsphere.io/en/terms)
 	* [MIT](https://opensource.org/licenses/MIT)
 	* SDISW Development License Agreement for Mendix Sample Apps EXTERNAL
 	* SDISW End-User License Agreement for Mendix Connectors and Widgets
 	* SDISW End-User License Agreement for Mendix Connectors and Widgets EXTERNAL
 	* SDISW End-User License Agreement for Mendix Sample Apps
```
Comment (KseniiaIankina) [inline content/en/docs/appstore/general/share-app-store-content.md]: Is it possible to add here 2 tables here . They are required for more detailed information on the licenses difference 
https://paper.dropbox.com/doc/License-Check--BgtW354EGB5cb_DSbe5N5kV6Ag-J7lRtFdRcOoVypy22ZTgg
OSS Licencee Requirements for Marketplace Components
SDISW and Mindsphere License Requirements for Marketplace Components

File: content/en/docs/appstore/general/share-app-store-content.md (hunk 3 above)
Comment (KseniiaIankina) [inline content/en/docs/appstore/general/share-app-store-content.md]: Mendix sends a notification to the owner of the outdated component and company admin, who then ..

File: content/en/docs/appstore/general/share-app-store-content.md [hunk 5]
```diff
@@ -256,7 +269,22 @@ To update content that has already been published, follow these steps:
...
+* Whether the component has been updated recently or not for ages (for example, it was published in 2016 and has not been updated since)
+* Whether it is being actively used or if it has limited usage or very few downloads, reviews, or ratings
+
+For a component that is outdated based on the above points, this is the review and remediation process: 
+
+1. Mendix sends a notification to the owner of the outdated component, who then has to submit an update within 30 days in order for their component to remain active on the Marketplace. This update needs to be based on support for an active version of Studio Pro.
+2. Mendix sends two reminders during these 30 days: the first on the 15th day, and the second on the 25th day.
+3. If the component owner is unable to make the required update within the stipulated timeframe, Mendix unpublishes their component from the Marketplace. Unpublishing means the component is not listed on the Marketplace, but a copy of the component remains in the database.
```
Comment (KseniiaIankina) [inline content/en/docs/appstore/general/share-app-store-content.md]: If the component owner or company admin is unable...

File: content/en/docs/appstore/general/share-app-store-content.md [hunk 6]
```diff
@@ -256,7 +269,22 @@ To update content that has already been published, follow these steps:
...
+4. If the owner wants to restore their unpublished component on the Marketplace, they make the required update and create a [Mendix Support](/developerportal/support/) request.
```
Comment (KseniiaIankina) [inline content/en/docs/appstore/general/share-app-store-content.md]: If the owner or company admin wants to restore..

File: content/en/docs/appstore/general/share-app-store-content.md [hunk 7]
```diff
@@ -121,22 +133,60 @@ Follow these steps to describe your content:
...
+#### 4.2.2 Providing License Details {#license)}
+
+Select the type of **License** you want applied to your app (if applicable). The lists and tables below describe the different considerations and requirements for these licenses.
+
+These are the OSS licenses and requirements:
+
+* [Apache V2](https://www.apache.org/licenses/LICENSE-2.0)
+* [GNU General Public License, version 3](https://www.gnu.org/licenses/gpl-3.0.en.html) – please note that everything licensed under GNU GPL is public and that GNU GPL has strong copyleft effect, which means that all consuming applications should make their code public as well; using this license is not recommended
+* [Mendix EULA](https://www.mendix.com/terms-of-use/) – this license is to be used when the component is released by Mendix
+* **Mendix Marketplace Content Master Terms**
```
Comment (Adam-Dupaski) [inline content/en/docs/appstore/general/share-app-store-content.md]: I see that this is now in the UI list of available licenses, can we describe/define this here?

File: content/en/docs/appstore/general/share-app-store-content.md [hunk 8]
```diff
@@ -121,22 +133,60 @@ Follow these steps to describe your content:
...
+* [MIT](https://opensource.org/licenses/MIT)
+
+| **License Name** | **Commercial Use allowed?** | **Component Code Needs to Be in Public Repo?** | **License Text Required with Copyright Information in Code & Distribution Artifact?** | **Can Modify?** (Modifications to the code should be mentioned) | **Can Consuming Apps Use Without Publicly Putting Their Code?** | **Notice Files Should Be Distributed with Artifact?** | **Original Component Source Code to Be Distributed with Consuming App?** | **Can Sub-License?** |
```
Comment (Adam-Dupaski) [inline content/en/docs/appstore/general/share-app-store-content.md]: "Without Publicly Putting Their Code" - the use of "Putting" is unclear here. Putting where? Can we replace with a better verb? Or with "Without Publicly Making Their Code Public" ?

File: content/en/docs/appstore/general/share-app-store-content.md [hunk 9]
```diff
@@ -121,22 +133,52 @@ Follow these steps to describe your content:
...
-	* SDISW End-User License Agreement for Mendix Connectors and Widgets EXTERNAL
-	* SDISW End-User License Agreement for Mendix Sample Apps
+
+#### 4.2.2 Providing License Details {#license)}
+
+Select the type of **License** you want applied to your app (if applicable). The lists and tables below describe the different considerations and requirements for these licenses.
+
+These are the OSS licenses and requirements available from the list on this page:
+
+* **Mendix Marketplace Content Master Terms**
```
Comment (Adam-Dupaski) [inline content/en/docs/appstore/general/share-app-store-content.md]: This is now in the UI list of available licenses, should we include this here in a table, and can we define/describe it?

File: content/en/docs/appstore/general/share-app-store-content.md [hunk 10]
```diff
@@ -121,22 +133,52 @@ Follow these steps to describe your content:
...
+| [MIT](https://opensource.org/licenses/MIT) | Make sure you add a specific *license.txt* file in your artifacts (meaning, in the *.mpk*). | {{< figure src="/attachments/appstore/general/share-app-store-content/check-mark.svg" >}} | {{< figure src="/attachments/appstore/general/share-app-store-content/check-mark.svg" >}} | {{< figure src="/attachments/appstore/general/share-app-store-content/check-mark.svg" >}} | {{< figure src="/attachments/appstore/general/share-app-store-content/check-mark.svg" >}} | {{< figure src="/attachments/appstore/general/share-app-store-content/check-mark.svg" >}} | {{< figure src="/attachments/appstore/general/share-app-store-content/check-mark.svg" >}} | {{< figure src="/attachments/appstore/general/share-app-store-content/cross-mark.svg" >}} | {{< figure src="/attachments/appstore/general/share-app-store-content/check-mark.svg" >}} |
+
+These are other OSS licenses and requirements:
```
Comment (Adam-Dupaski) [inline content/en/docs/appstore/general/share-app-store-content.md]: These license types aren't available in the UI list, so I've put them in a separate section. I am not sure how the user would use these for their component, if they're not available in the list? How should we position these?

File: content/en/docs/appstore/general/share-app-store-content.md (hunk 9 above)
Comment (KseniiaIankina) [reply content/en/docs/appstore/general/share-app-store-content.md]: Indeed, but it is only valid for our Mendix products as of now. So we will introduce the filtering and later only Mendix employees would see this option. So maybe to remove it from here ?

Comment (KseniiaIankina) [reply content/en/docs/appstore/general/share-app-store-content.md]: Or can we add to the table similar is Mendix EULA 
"This license is to be used when the component is released by Mendix" and with the same ticks ?

Comment (Adam-Dupaski) [reply content/en/docs/appstore/general/share-app-store-content.md]: If only Mendix employees will see this, then would be better to handle via internal process, governance, and internal documentation. If it's an internal thing, it shouldn't be on customer-facing docs. I can remove it - do you agree?

Comment (KseniiaIankina) [reply content/en/docs/appstore/general/share-app-store-content.md]: Yes, you are right , let's remove.

Comment (KseniiaIankina) [reply content/en/docs/appstore/general/share-app-store-content.md]: Yes, you are right , let's remove.

File: content/en/docs/appstore/general/share-app-store-content.md (hunk 10 above)
Comment (KseniiaIankina) [reply content/en/docs/appstore/general/share-app-store-content.md]: Hi Adam, we have the discussions regarding the following licenses to have the final decision if we allow LGPL or not and also discuss which licenses to add in the UI. Is it possible to push the release of the docu without this table and  to add this table in the next PR ? That way we will have some time to finilise , but already commit whatever is final at this moment. Thank you in advance !

Comment (Adam-Dupaski) [reply content/en/docs/appstore/general/share-app-store-content.md]: Good idea @KseniiaIankina I've removed this table, will publish everything else in this PR, and I'll create a new PR for analysis of what was removed here.

**Pull request merged**